ui/src/app/job-list/table/table.component.ts

```diff
@@ -90,7 +90,7 @@ export class JobsTableComponent implements OnInit {
     return job.status == JobStatus.Submitted || job.status == JobStatus.Running;
   }
 
-  canAbortAnySelected(): boolean {
+  showSelectionBar(): boolean {
```

Comment:
The ticket also mentions changing this logic to do a grayed out abort instead of not showing the controls; this could be a follow-up change though, up to you.

Reply:
I like that solution, added.